assisted_deployment_utils.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+set -o pipefail
+source common.sh
+source logging.sh
+
+TEST_INFRA_DIR=$WORKING_DIR/test-infra
+export TEST_INFRA_BRANCH=${TEST_INFRA_BRANCH:-master}
+export ASSISTED_NETWORK=${ASSISTED_NETWORK:-"test-infra-net"}
+export INSTALL=${INSTALL:-"y"}
+export WAIT_FOR_CLUSTER=${WAIT_FOR_CLUSTER:-"y"}
+export INSTALLER_IMAGE=${INSTALLER_IMAGE:-}
+export SERVICE=${SERVICE:-}
+
+
+function destroy_assisted_nodes(){
+  echo "Destroying assisted_deployment vms"
+  run_assited_command "destroy_nodes"
+}
+
+function run_assited_command () {
```

Comment:
This typo is likely to lead to confusing errors when someone spells the function correctly and it doesn't work - can you sed this to correct the spelling please?